roles/edpm_bootstrap/tasks/download_cache.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Download needed packages
+  ansible.builtin.dnf:
```

Comment:
should we add retries?

Reply:
Mmmh, given that it could have sense and I'm not against it...we didn't have similar behavior for packages installation tasks, why we should need it for download? 

Do you want to prevent the playbook failure for temporary network issues or something like that?

Reply:
Exactly! During the adoption hackathon I've seen people having network issues, and currently, when a job fails, it takes 10 minutes to retry: https://github.com/openstack-k8s-operators/edpm-ansible/pull/314

Reply:
yeah,i like the idea, we could implement it both for download and install tasks (if download_cache service isn't triggered, downloads are performed during installation tasks to avoid errors).

Reply:
@fao89 take a look now :)